central/billingmetrics/store/postgres/store.go

```diff
@@ -0,0 +1,150 @@
+package postgres
+
+// Code is not generated by pg-bindings generator. DO EDIT.
```

Comment:
Could you add some more context around your question in slack (https://redhat-internal.slack.com/archives/C04SZM93MUK/p1690216132727209) and what the outcome / decision is? I feel like the discussion lacked some details and was therefore a bit surface level.